packages/portfolio-contract/test/snapshots/portfolio.flows.test.ts.md

```diff
@@ -707,12 +806,12 @@ Generated by [AVA](https://avajs.dev).
         'published.ymax0.portfolios.portfolio1.positions.position1',
         {
           accountId: 'eip155:1:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
-          protocol: 'Compound',
+          protocol: 'Aave',
         },
       ],
     ]
 
-## handle failure in localTransfer from seat to local account
+## open portfolio with Compound position
```

Comment:
ugh. the snapshots got completely out of sync.

It would save me review time if, instead of changing one of the earlier tests, you'd copy it to the end as a new test and then modify it.